### DISCUSSION TITLE
Multiple collection reporting

### DISCUSSION THREAD

**Discussion by lowtechninja** [NUWCDIVNPT/stig-manager#515] [Ideas and Enhancements]
It would be nice to pull reports from multiple collections

**Comment by cd-rite** [comment]
Thanks @lowtechninja 
As I mentioned in the other thread, I think this feature would satisfy at least some of the need for your other suggestion of sub-collections. 
As a bonus, this approach would probably be much easier to implement than a sort of nested Collectionception structure, which would probably require a lot of access control considerations.  

This feature could probably be accomplished without any API changes at all, but just a client that was able to fetch the statistics available for each Collection the user selects (and that they have access to), and provide a summed view.

The only issue I see with this at the moment is that Assets must be unique within Collections, but we do not enforce that requirement across different Collections.  This, again, could be analyzed and flagged at the client presentation level as a possible issue for a User to decide how to interpret, but would be .a complicating factor. 

Either way, we think the reporting area has a lot of potential for expansion. It's also one of the easier ways for others to expand on the project, if you have developer resources that could create another client to hit our API.  We will also definitely keep this in mind for our next UI feature expansion. 

Thanks!
